src/core/localization/translations/en/common.json

```diff
@@ -341,6 +341,7 @@
     "konturTheme": "Kontur",
     "HOTTheme": "HOT",
     "userBio(about)": "Bio",
+    "userBioPlaceholder" :"Tell us a bit about yourself:\n  •  your current role\n  •  professional skills\n  •  interests\n  •  industry\n  •  objectives or challenges you tackle\nThis bio helps customize and personalize your experience.",
```

Comment:
Please use key name with **snake_case** instead of camelCase (profile keys should be fixed later at some point, for now let's create new ones using snake case)

Reply:
Thank you. fixed